### ISSUE TITLE
Calendar Customizer for Users

### ISSUE THREAD

**Issue by niccokunzmann** [fullcalendar/fullcalendar#4523]
Hi,

I am writing a project which makes it easy for people to customize their calendar and embed it into a website, currently hosted at https://open-web-calendar.herokuapp.com/.

To me, it looks like fullcalendar is a) great and open-source and b) for developers.
I would like to drive the usage threshold a bit down by providing a customization interface for users who are not afraid of an iframe or links but would not code.

So, this issue is about your plans and maybe embedding fullcalendar into the app I wrote.

What are your thoughts?

**Comment by acerix** [comment]
I'm not sure what you are asking, the calendar is MIT licensed so it's fine to use in your project.

**Comment by niccokunzmann** [comment]
@acerix Thanks for your answer. My question is if I overlooked something: 

1. Is there a customizer like open web calendar for this calendar? 
2. Is this calendar targeted at developers at the current state?

If yes, https://github.com/niccokunzmann/open-web-calendar/issues/38 would bring it to a wider audience. It is more a community question than a code/legal question but I did not find a place to discuss this other than here.
